cost-analyzer/templates/cost-analyzer-network-policy-template.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.networkPolicy -}}
+{{- if .Values.networkPolicy.costAnalyzer.enabled -}}
```

Comment:
Just a quick note that for backwards compatibility we check every level of the object so we'd also want a check for 
.Values.networkPolicy.constAnalyzer, but I can add it for you.